qa/logging-config/src/test/java/org/elasticsearch/common/logging/JsonLoggerTests.java

```diff
@@ -203,22 +202,17 @@ public void testDeprecatedMessageWithoutXOpaqueId() throws IOException {
                     hasEntry("cluster.name", "elasticsearch"),
                     hasEntry("node.name", "sample-name"),
                     hasEntry("message", "deprecated message1"),
-                    not(hasKey("x-opaque-id"))
-                ),
-                allOf(
-                    hasEntry("type", "deprecation"),
-                    hasEntry("log.level", "DEPRECATION"),
-                    hasEntry("log.logger", "deprecation.test"),
-                    hasEntry("cluster.name", "elasticsearch"),
-                    hasEntry("node.name", "sample-name"),
-                    hasEntry("message", "deprecated message2"),
+                    hasEntry("data_stream.type", "logs"),
+                    hasEntry("data_stream.datatype", "deprecation"),
+                    hasEntry("data_stream.namespace", "elasticsearch"),
+                    hasEntry("ecs.version", DeprecatedMessage.ECS_VERSION),
```

Comment:
Should we check for the presence of `key` here?

Reply:
yes, will add - missed it